### ISSUE TITLE
(WEB REPORT BY: Anonymous REMOTE: ) Item Disappeared after Uncuffing

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#5150]
>Revision (Should be above if you're viewing this from ingame!)

>General description of the issue
Any item placed in hands while handcuffed disappeared into player's hand once uncuffed.

>What you expected to happen
When I placed the mining bag in the player's hands, it should remain there even after being uncuffed.

>What actually happened
I'm assuming after being uncuffed, everything in the player's hands must've been deleted.

>Steps to reproduce if possible
Handcuff player. (Mining bag.)
Place item in hand of player.
Remove handcuffs.

**Comment by Drag3ndz** [comment]
I was the one who reported this issue if you have any questions.

**Comment by ComicIronic** [comment]
How were you placing an item in their hands?

**Comment by Drag3ndz** [comment]
As security, I handcuffed a skeleton engineer and strapped him to a chair. After stripping their stuff and confirming they did nothing wrong (security failed to give me a reason to lock him up) I started putting stuff back on him. First the backpack, then the engineering hardsuit, and finally the miner bag on his right hand while he was cuffed. I then moved him to the front of security and removed his handcuffs.
